data/json/items/ranged.json

```diff
@@ -4213,6 +4213,38 @@
     "valid_mod_locations": [[ "accessories", 4 ],[ "grip", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
   },
   {
+    "id": "m41a_pulse_rifle",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "teal",
+    "name": "M41A pulse rifle",
```

Comment:
Perhaps it should be called something like 'EMP rifle' or something. I know the current name would confuse the heck out of me, because the M41A is a pretty specific weapon.